azure-stack/hci/concepts/other-security-features.md

```diff
@@ -0,0 +1,518 @@
+---
+title: Other security features for Azure Stack HCI (preview)
+description: Learn about security features for Azure Stack HCI (preview) security information and event management (SIEM).
+author: alkohli
+ms.author: alkohli
+ms.topic: conceptual
+ms.service: azure-stack
+ms.subservice: azure-stack-hci
+ms.date: 10/27/2023
+---
+
+# Other security features for Azure Stack HCI (preview)
+
+[!INCLUDE [applies-to](../../includes/hci-applies-to-supplemental-package.md)]
+
+This article describes new components in the security layer of the Azure Stack HCI solution for security information and event management (SIEM).
+
+## Renamed local built-in users
+
+We renamed the local built-in users for the `RID 500 local administrator` and `RID 501 guest` accounts. The new names and status of those accounts are as follows:
+
+|Name |Enabled |Description |
+|-----|-----|-----|
+|ASBuiltInAdmin |True |Built-in account for administering the computer/domain |
+|ASBuiltInGuest |False |Built-in account for guest access to the computer/domain |
+
+> [!Important]
+> We recommend that you create your own local administrator account, and also that you disable the well-known `RID 500` user account.
+
+The `Guest` account is protected by the security baseline drift control mechanism. For more information, see [Security baseline settings for Azure Stack HCI (preview)](/azure-stack/hci/concepts/secure-baseline).
+
+We also added two net new components to Azure Stack HCI in 23H2 release, as follows:
+
+- Secrets management engine.
+- SIEM Syslog forwarder agent.
+
+## Secrets management engine
```

Comment:
Consider rephrasing the heading text to: 'Manage secrets in Azure Stack HCI'. 

This edit requires SME validation.